coreclient/src/conversations/messages/mod.rs

```diff
@@ -93,11 +98,12 @@ impl ConversationMessageId {
     }
 }
 
-#[derive(PartialEq, Debug, Clone, Serialize, Deserialize)]
+#[derive(PartialEq, Debug, Clone)]
 pub struct ConversationMessage {
     pub(super) conversation_id: ConversationId,
     pub(super) conversation_message_id: ConversationMessageId,
     pub(super) timestamped_message: TimestampedMessage,
+    pub(super) status: EnumSet<MessageStatusBit>,
```

Comment:
Why do we use an EnumSet? Could we just store something like max([MessageStatusBit]) ?

Reply:
I just looked into the draft again and it says "The status of each message can be one of the following values: [...]". I can work on this tomorrow.